### DISCUSSION TITLE
QALY loss due to mortality, stratified by clinical risk group

### DISCUSSION THREAD

**Discussion by Neil-W1** [HTAhackathon2024/.github#21] [Ideas]
### What is the problem we're trying to solve

I work a lot on models looking at cost-effectiveness of vaccination programmes, which are usually stratified by clinical risk group.

However, we generally tend to assign the 'low risk' and 'high risk' groups the same QALY loss due to mortality, which would overstate the cost-effectiveness of a programme targeted at the high risk.

So it would be nice to calculate QALY loss due to mortality, stratified by clinical risk group.

### How will we tackle the problem?

1. Work out if this has already been done. @n8thangreen I see that your [QALY package](https://github.com/Health-Economics-in-R/QALY?tab=readme-ov-file) mentions co-morbidities, so not sure?
2. If not, come up with a method for doing it
3. Collect data on co-morbidities that can be used (we probably just want to focus on a few)
4. Modify the QALY calculation

### Which tools will we use?

It could be an extension to the `{QALY}` package.

### Previous work

This [paper](https://onlinelibrary.wiley.com/doi/10.1002/hec.4208) proposes a method for including co-morbidities in QALY calculations. It has a couple of hypothetical parameters that would somehow need to be estimated based on real data.

**Comment by n8thangreen** [comment]
I think we could take the method from the paper and add it to `{QALY}`. This package is very buggy so happy to change how its currently written!
